ginga/misc/plugins/Cuts.py

```diff
@@ -221,6 +235,29 @@ def build_gui(self, container):
         self.select_cut(self.cutstag)
         self.gui_up = True
 
+    def build_axes(self, vbox):
+        fr = Widgets.Frame("Axes")
+        hbox = Widgets.HBox()
+        hbox.set_border_width(4)
+        hbox.set_spacing(1)
+
+        image = self.fitsimage.get_image()
+        if image is not None:
+            # Add Checkbox widgets
+            for i in xrange(0, len(image.get_mddata().shape)+1):
+                name = 'NAXIS%d' % i
+                chkbox = Widgets.CheckBox(name)
+                self.axes_chkboxes.append(Bunch.Bunch(widget=chkbox, pos=i, state=False))
+                hbox.add_widget(chkbox)
+
+            # Add callbacks
+            for chkbox in self.axes_chkboxes:
+                cbox = chkbox.get('widget')
+                cbox.add_callback('activated', lambda w, tf: self.axis_toggle_cb(w, tf, chkbox))
```

Comment:
@ejeschke I am trying to add check boxes according to the number of axes so that multiple axes can be used for slicing. What I can't figure out is how to add a callback for each checkbox.